src/main/java/com/example/demo/DemoApplication.java

```diff
@@ -1,27 +1,29 @@
 package com.example.demo;
 
+import com.example.demo.matcher.InitialiseFakeMarket;
+import com.example.demo.matcher.Matcher;
 import com.example.demo.security.service.UserService;
 import com.example.demo.security.userInfo.AppUser;
 import com.example.demo.security.userInfo.Role;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.CommandLineRunner;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.annotation.Bean;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
-import org.springframework.security.crypto.password.PasswordEncoder;
 
+import javax.persistence.Tuple;
 import java.util.ArrayList;
 
-@SpringBootApplication
+@SpringBootApplication @AllArgsConstructor
```

Comment:
Do you still need this annotation? I thought you only needed because you were autowiring the matcher?